using_images/xpaas_images/eap.adoc

```diff
@@ -121,28 +112,72 @@ If you want the JBoss EAP xPaaS image to use a different JDK version than the de
   "value": "/usr/lib/jvm/java-1.7.0"
 }
 ----
++
 
-=== Using a Modified JBoss EAP xPaaS Image
 
-An alternative method is to make changes to the image, and then use that modified image in OpenShift.
+== Getting Started Using xPaaS JBoss EAP Images
 
-The JBoss EAP configuration file that OpenShift uses inside the JBoss EAP xPaaS image is *_EAP_HOME/standalone/configuration/standalone-openshift.xml_*, and the JBoss EAP startup script is *_EAP_HOME/bin/openshift-launch.sh_*.
+=== Configure the xPaaS JBoss EAP Images
 
-You can run the JBoss EAP xPaaS image in Docker, make the required configuration changes using the JBoss EAP Management CLI (*_EAP_HOME/bin/jboss-cli.sh_*), and then commit the changed container as a new image. You can then use that modified image in OpenShift.
+You can change the configuration for the xPaaS JBoss EAP images by either using the S2I (Source to Image) templates, or by using a modified xPaaS JBoss EAP image. Red Hat recommends using the S2I method to configure the xPaaS JBoss EAP image.
 
-[IMPORTANT]
-It is recommended that you do not replace the OpenShift placeholders in the JBoss EAP xPaaS configuration file, as they are used to automatically configure services (such as messaging, datastores, HTTPS) during a container's deployment. These configuration values are intended to be set using environment variables.
+Configure the xPaaS JBoss EAP Image using the S2I Templates
+The recommended method to run and configure the xPaaS JBoss EAP image is to use the OpenShift S2I process together with the application template parameters and environment variables.
 
 [NOTE]
-Ensure that you follow the   link:../../creating_images/guidelines.html[guidelines for creating images].
+====
+The variable `*EAP_HOME*` is used to denote the path to the JBoss EAP installation. Replace this variable with the actual path to your JBoss EAP installation.
+====
+
+The S2I process for the xPaaS JBoss EAP image works as follows:
+
+. If there is a *_pom.xml_* file in the source repository, a Maven build is triggered with the contents of `*$MAVEN_ARGS*` environment variable. By default the package goal is used with the OpenShift profile, including the system properties for skipping tests (`*-DskipTests*`) and enabling the Red Hat GA repository (`*-Dcom.redhat.xPaaS.repo.redhatga*`). The results of a successful Maven build are copied to `*EAP_HOME/standalone/deployments*`. This includes all JAR, WAR, and EAR files from the directory within the source repository specified by `*$ARTIFACT_DIR*` environment variable. The default value of `*$ARTIFACT_DIR*` is the target directory.
```

Comment:
Yeah, I'll be honest, this is inherited stuff and I only have a basic understanding of Maven and goals within Maven projects, and not really much about these specific variables. I've cleaned up the language to be a bit clearer, hopefully. Let me know if you think it requires further work.